src/states/stay/CountdownToStay.java

```diff
@@ -1,7 +1,60 @@
 package states.stay;
 
-import states.Countdown;
+import events.TimerRanOut;
+import events.TimerTicked;
+import events.TimerUp;
+import states.SecurityContext;
+import states.SecurityState;
+import timer.Notifiable;
+import timer.Timer;
 
-public class CountdownToStay extends Countdown {
+public class CountdownToStay extends SecurityState implements Notifiable {
```

Comment:
Countdown already extends security state, unless you opted to take a different approach that doesn't rely on the inheritance from Countdown

Reply:
That's exactly what I did. I'm glad you went your way, because we can compare functionalities and how they work.